samples/sensor/mpr/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
please move the sample to a separate commit. you can keep it in the same PR though

Reply:
Ok, done.